cmd/syft/internal/options/license.go

```diff
@@ -14,15 +17,25 @@ var _ interface {
 } = (*licenseConfig)(nil)
 
 func (o *licenseConfig) DescribeFields(descriptions clio.FieldDescriptionSet) {
-	descriptions.Add(&o.IncludeUnknownLicenseContent, `include the content of a license in the SBOM when syft
+	descriptions.Add(&o.IncludeUnknownLicenseContent, `deprecated: include the content of a license in the SBOM when syft
 cannot determine a valid SPDX ID for the given license`)
+	descriptions.Add(&o.IncludeLicenseContent, `include the content of licenses in the SBOM for a given syft scan; valid values are: all, unknown, or none; default is none`)
 	descriptions.Add(&o.LicenseCoverage, `adjust the percent as a fraction of the total text, in normalized words, that
 matches any valid license for the given inputs, expressed as a percentage across all of the licenses matched.`)
 }
 
+func (o *licenseConfig) PostLoad() error {
+	if o.IncludeUnknownLicenseContent && (o.IncludeLicenseContent == "none" || o.IncludeLicenseContent == "all") {
+		log.Warnf("ignoring deprecated option: include-unknown-license-content=%t infavor of include-license-content=%s", o.IncludeUnknownLicenseContent, o.IncludeLicenseContent)
+		log.Warnf("please update your configuration to use include-license-content option")
```

Comment:
this shouldn't be logging, but instead raise an error. If someone has conflicting configuration we should stop and not produce a result until the configuration is sorted.